core/trino-main/pom.xml

```diff
@@ -535,10 +535,53 @@
         <resources>
             <resource>
                 <directory>src/main/resources</directory>
+                <includes>
+                    <include>oauth2/</include>
+                    <include>webapp/</include>
+                    <include>webapp-preview/dist/</include>
+                </includes>
                 <excludes>
                     <exclude>**/node_modules/**</exclude>
                 </excludes>
             </resource>
         </resources>
+        <plugins>
+            <plugin>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
+                <version>${dep.frontend-maven-plugin.version}</version>
+                <configuration>
+                    <nodeVersion>v20.15.1</nodeVersion>
+                    <npmVersion>10.8.2</npmVersion>
```

Comment:
It would be better to extract these versions to a property, either in this module or in the parent pom

Reply:
fixed